src/battle/overlay_12_0224E4FC.c

```diff
@@ -3329,7 +3485,21 @@ int GetBattlerLearnedMoveCount(BattleSystem *bsys, BATTLECONTEXT *ctx, int battl
     return cnt;
 }
 
-extern u16 sSoundMoves[12];
+//BUG: There isn't a reason why this shouldn't be const, but it's located in .data and not .rodata
```

Comment:
not *really* a bug, just a poor choice by GF

Reply:
I think it was just someone forgetting to write 'cosnt'